hotline/species/tests/test_forms.py

```diff
@@ -0,0 +1,52 @@
+from django.test import TestCase
+from django.test.client import RequestFactory
+from model_mommy.mommy import make
+
+from hotline.users.models import User
+
+from ..forms import SpeciesSearchForm
+from ..models import Species
+
+
+class SpeciesFormsTest(TestCase):
+    """
+    Some unit tests to ensure forms in the species app work as they should.
+
+    """
+    def setUp(self):
+        super(SpeciesFormsTest, self).setUp()
+        make(Species, name="stuff")
+        self.user = User(first_name="foo", last_name="bar", email="foobar@example.com", is_staff=True)
+        self.user.set_password("foobar")
+        self.user.save()
+        self.client.login(username="foobar@example.com", password="foobar")
+
+    def test_get_queryset(self):
```

Comment:
One thing I'm starting to do more in my tests is use really, really, really verbose function names, like `test_get_queryset_returns_all_species_from_model`. That way, you don't (always) have to write the clarifying docstring.